src/shogun/modelselection/GradientModelSelection.cpp

```diff
@@ -88,12 +88,14 @@ class GradientModelSelectionCostFunction: public FirstOrderCostFunction
 		SG_ADD((CSGObject **)&m_obj, "GradientModelSelectionCostFunction__m_obj",
 			"obj in GradientModelSelectionCostFunction", MS_NOT_AVAILABLE);
 		m_func_data = NULL;
+        /*
```

Comment:
@karlnapf need your comments here. They were registered by value which is not good.